lib/getSiteVersion.ts

```diff
@@ -0,0 +1,8 @@
+const SITE_VERSION_HACKATHON_2023 = "hackathon-2023";
+
+const getSiteVersion = (): string | undefined => {
```

Comment:
This is kind of an unneeded function in one way as it's a one liner but I think it's a good abstraction if we want to change the way we grab this information.

Reply:
I like it!